### DISCUSSION TITLE
每次进去都是同一个世界吗

### DISCUSSION THREAD

**Discussion by Xiaohaoziiiii** [paternostrox/Minecraft4Unity#5] [General]
我退出又重进过多次，都是同一个世界

**Comment by paternostrox** [comment]
This is expected, the world is generated according to the player position, it will generate the same world as long the player starts in the same position. Open the game scene and you will find a player gameobject, if you want it to start in a different position, simply change it's position and save the scene before starting the game. Or you could write a simple script to change the position randomly.
